C_CPP/Algorithmic Approaches/Sorting/sorting.md

```diff
@@ -0,0 +1,252 @@
+## what is sorting?
```

Comment:
Vandana, Some very basic things like sentence starts with a "Capital alphabet".
Update this in all your headings.